llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -2181,6 +2187,10 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
     return ConstantDataVector::get(IRB.getContext(), OrderingTable);
   }
 
+  void instrument(Instruction &I) {
```

Comment:
Why don't we just call `InstVisitor<MemorySanitizerVisitor>::visit` in the loop?